Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: deploy deploy-staging sync-on-prod fetch-nginx-config
+.PHONY: deploy deploy-staging sync-on-prod fetch-nginx-config clipping-service
```

Should the other new targets also be added to the `.PHONY` list?

```suggestion
.PHONY: deploy deploy-staging sync-on-prod fetch-nginx-config clipping-service env-clip ckan-dev
```